Backend/src/main/java/ca/mcgill/cooperator/controller/CourseController.java

```diff
@@ -102,27 +95,4 @@ public CourseDto deleteCourse(@PathVariable int id) {
 
         return ControllerUtils.convertToDto(deletedCourse);
     }
-
-    @ExceptionHandler(RuntimeException.class)
-    public final ResponseEntity<Exception> handleAllExceptions(RuntimeException ex) {
-        return new ResponseEntity<Exception>(ex, HttpStatus.INTERNAL_SERVER_ERROR);
-    }
-
-    /**
-     * Delete an existing Course
-     *
-     * @param list of CourseOfferingDto objects
-     * @return list of CourseOffering objects
-     */
-    private List<CourseOffering> convertCourseOfferingListToDomainObject(
-            List<CourseOfferingDto> coDtos) {
-        List<CourseOffering> cos = new ArrayList<>(coDtos.size());
-        for (CourseOfferingDto coDto : coDtos) {
-            if (coDto != null) {
-                CourseOffering co = courseOfferingService.getCourseOfferingById(coDto.getId());
-                cos.add(co);
-            }
-        }
-        return cos;
```

Comment:
why did you delete this?

Reply:
I just moved this to `ControllerUtils` so that all of these types of methods are in one place

Reply:
@kraglalbert you should make sure it still works because i had it there when i wrote this but the courseOfferingService was always null even though it was auto wired (i think bc the controller utils class is never defined with the auto wired thing in the classes that use it)